parsers/OPENNEM.py

```diff
@@ -87,23 +134,32 @@ def fetch_production(zone_key=None, session=None, target_datetime=None, logger=l
     objs = [{
         'datetime': arrow.get(dt.to_pydatetime()).datetime,
         'production': {  # Unit is MW
-            'coal': sum_vector(row, ['COAL_BLACK', 'COAL_BROWN']),
-            'gas': sum_vector(row, ['GAS_CCGT', 'GAS_OCGT', 'GAS_RECIP', 'GAS_STEAM']),
-            'oil': sum_vector(row, ['DISTILLATE']),
-            'hydro': sum_vector(row, ['HYDRO']),
-            'wind': sum_vector(row, ['WIND']),
-            'biomass': sum_vector(row, ['BIOENERGY_BIOGAS']),
+            'coal': sum_vector(row, OPENNEM_PRODUCTION_CATEGORIES['coal']),
+            'gas': sum_vector(row, OPENNEM_PRODUCTION_CATEGORIES['gas']),
+            'oil': sum_vector(row, OPENNEM_PRODUCTION_CATEGORIES['oil']),
+            'hydro': sum_vector(row, OPENNEM_PRODUCTION_CATEGORIES['hydro']),
+            'wind': sum_vector(row, OPENNEM_PRODUCTION_CATEGORIES['wind']),
+            'biomass': sum_vector(row, OPENNEM_PRODUCTION_CATEGORIES['biomass']),
             # We here assume all rooftop solar is fed to the grid
             # This assumption should be checked and we should here only report
             # grid-level generation
-            'solar': sum_vector(row, ['SOLAR_UTILITY', 'SOLAR_ROOFTOP']),
+            'solar': sum_vector(row, OPENNEM_PRODUCTION_CATEGORIES['solar']),
         },
         'storage': {
             # opennem reports charging as negative, we here should report as positive
```

Comment:
do we? i thought OpenNEM reports charging as positive.

Reply:
I don't know :D Could you let me know just to confirm?

Reply:
friendly ping here

Reply:
^ @nc9

Reply:
Checked with @nc9 - confirmed!